srcpkgs/grandorgue/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'grandorgue'
+pkgname=grandorgue
+version=3.13.0
+revision=1
+archs="x86_64"
+build_style="cmake"
+configure_args="-DCMAKE_BUILD_TYPE=Release -G Ninja"
```

Comment:
neither of these need to be set. void sets the build type to None and the generator to ninja by default in `common/build-style/cmake.sh`